strawberryfields/decompositions.py

```diff
@@ -476,6 +476,190 @@ def rectangular_symmetric(V, tol=1e-11):
     return new_tlist, new_diags, None
 
 
+def BS_mat(theta):
+    r"""Beam splitter matrix as implemented in hardware
+    """
+    mat = np.array([[np.cos(theta), 1j*np.sin(theta)], [1j*np.sin(theta), np.cos(theta)]])
```

Comment:
Just curious, off the top of my head, the default SF beamsplitter is the following?
```python
np.array([[np.cos(theta), -np.exp(-1j*phi)*np.sin(theta)], [np.exp(1j*phi)*np.sin(theta), np.cos(theta)]])
```
So isn't this the same as the default SF beamsplitter, just with phi->pi/2? Although I might be misremembering the SF default.